sklearnex/linear_model/coordinate_descent.py

```diff
@@ -15,3 +15,12 @@
 # ===============================================================================
 
 from daal4py.sklearn.linear_model import ElasticNet, Lasso
+from onedal._device_offload import support_usm_ndarray
+
+ElasticNet.fit = support_usm_ndarray(queue_param=False)(ElasticNet.fit)
+ElasticNet.predict = support_usm_ndarray(queue_param=False)(ElasticNet.predict)
+ElasticNet.score = support_usm_ndarray(queue_param=False)(ElasticNet.score)

```

Comment:
JFYI: __doc__ is saved during this decoration.

Reply:
Yes, that is right. Therefore unnecessary doc assignments are not added.